### ISSUE TITLE
Using "ES6 Import" instead of require lead to TypeError after compilation

### ISSUE THREAD

**Issue by Geolim4** [twigjs/twig.js#830]
Hello,

I tried to use 

``` ts
import Twig from 'twig';
```

instead of

``` ts
const Twig = require('twig').twig;
```

And the build passes successfully but when I'm loading the app in the browser, I got this error:
**TypeError: he() is not a function** (_he_ being the Twig import)

I may missed something but I was wondering if twigjs is ES6-compatible ?

**Comment by willrowe** [comment]
No, it does not currently support ES modules. This is a duplicate of #622.
